tools/FileTestDefs.sh

```diff
@@ -38,9 +38,34 @@ staged_grep() {
 # Utility function for reporters that enables lots of decorators in grep
 # Works like staged_grep
 pretty_grep() {
-    echo -n -e "\033[0;35m${@: -1}\033[0m:"
-    git show ":./${@: -1}" | \
-        GREP_COLOR='1;37;41' grep -n $color_option "${@:1:$(($#-1))}"
+    local -a args
```

Comment:
args would be easier to understand if called something along the lines of `grep_options` or `grep_args`